frontend/apps/filemanager/filemanager.lua

```diff
@@ -735,21 +736,27 @@ function FileManager:tapPlus()
                     text = _("Import files here"),
                     enabled = Device:isValidPath(self.file_chooser.path),
                     callback = function()
-                        UIManager:close(self.file_dialog)
+                        UIManager:close(plus_dialog)
                         Device.importFile(self.file_chooser.path)
                     end,
                 },
             })
         end
+
+        local extract_button = self.coverbrowser and self.coverbrowser:genExtractBookInfoButton(close_dialog_callback)
```

Comment:
(Not fond of seeing named references to a plugin in core - but you have already done that a lot, so ok :))

Reply:
I do hope one day you will agree it deserves to be moved to the core, it's the basis of the library.

Reply:
Besides of the view, the database part is important. As we have measured, the sql request is faster than doc_settings opening.

Reply:
I support someone other than me doing it. :innocent:

Reply:
> I do hope one day you will agree it deserves to be moved to the core

I have nothing against that (see https://github.com/koreader/koreader/issues/8472#issuecomment-1554638967) - as long as the database is still considered just a cache and trashable (and disable'able if possible).

> I support someone other than me doing it.

It's huge work, so: me too :)

Reply:
I think they are giving you the go ahead... @hius07